src/main/java/com/github/maleksandrowicz93/cqrsdemo/student/port/outgoing/Student.java

```diff
@@ -0,0 +1,27 @@
+package com.github.maleksandrowicz93.cqrsdemo.student.port.outgoing;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+import java.time.LocalDate;
+import java.util.UUID;
+
+@ToString
+@Getter
+@Setter
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class Student {
```

Comment:
czym jest ta klasa? wygląda jak główny obiekt domenowy, ale nie ma żadnych metod + jest w pakiecie `port.outgoing`

wydaje się, że to tutaj powinno być gros "logiki", a obecnie jest to zwykły DTO